pkg/genericapiserver/server/storage_factory.go

```diff
@@ -30,6 +30,15 @@ import (
 	"github.com/golang/glog"
 )
 
+// Backend describes the storage servers, the information here should be enough
+// for health validations
+type Backend struct {
+	Server string
+	// TLS credentials
+	KeyFile  string
```

Comment:
Are these really file paths rather than the contents of the key/cert files? Seems odd to pass around file paths.